<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>yexiaopeng</author>
 <class>QSocketTools</class>
 <widget class="QDialog" name="QSocketTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSocketTools</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1001</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tcp_client">
    <attribute name="title">
     <string>tcp_client</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>1011</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 238);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_save_receive_data">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tcpclient_local_port">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>170</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tcpclient_server_add">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tb_tcpclient_receivedata">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>701</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_clear_receive_data">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>380</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_show_receive_data_hex">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>260</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Hex Show</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tcpclient_local_add">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_show_receive_data_date">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Date</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tcpclient_server_port">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_clear_send_data">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>clear Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_pause_receive_data">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause Show</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Local  Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_connect">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monaco</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_tcpclient_send_data">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>send Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Server Add</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_tcpclient_send_data">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>671</width>
       <height>72</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="udp">
    <attribute name="title">
     <string>udp</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tb_udp_receivedata">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>501</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_show_receive_data_hex">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>260</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Hex Show</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_pause_receive_data">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause Show</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_clear_send_data">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>clear Data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_show_receive_data_date">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Date</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_connect">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_clear_receive_data">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>380</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_udp_local_port">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>170</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Local  Add</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_udp_send_data">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>501</width>
       <height>72</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_udp_targe_add">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_save_receive_data">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_udp_local_add">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_udp_send_data">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>send Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿等宽微米黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>targe Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_udp_targe_port">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
